packages/calypso-build/webpack/sass.js

```diff
@@ -4,19 +4,18 @@
 const FilterWarningsPlugin = require( 'webpack-filter-warnings-plugin' );
 const MiniCssExtractPluginWithRTL = require( 'mini-css-extract-plugin-with-rtl' );
 const WebpackRTLPlugin = require( 'webpack-rtl-plugin' );
-const path = require( 'path' );
 
 /**
  * Return a webpack loader object containing our styling (Sass -> CSS) stack.
  *
  * @param  {Object}    _                              Options
- * @param  {boolean}   _.preserveCssCustomProperties  whether Custom Properties and properties using them should be preserved in their original form
  * @param  {string[]}  _.includePaths                 Sass files lookup paths
  * @param  {string}    _.prelude                      String to prepend to each Sass file
+ * @param  {string}    _.postCssConfigPath            Path to PostCSS config file
  *
  * @return {Object}                                   webpack loader object
  */
-module.exports.loader = ( { preserveCssCustomProperties, includePaths, prelude } ) => ( {
+module.exports.loader = ( { includePaths, prelude, postCssConfigPath } ) => ( {
```

Comment:
If we are making the PostCSS plugin completely customizable from outside, we should also add the `postCssConfigContext` option and pass it as [`context`](https://github.com/postcss/postcss-loader#context-ctx).

Reply:
Okay, makes sense to reflect https://github.com/postcss/postcss-loader#config. Personally, I would've gone with YAGNI here, but I'm fine with adding this for completeness' sake.

Reply:
e99e44134cc7417dfbe07598669645e57602b3be